.eslintrc.json

```diff
@@ -1,6 +1,7 @@
 {
   "root": true,
   "plugins": ["simple-import-sort", "unused-imports"],
+  "ignorePatterns": ["vite.config.ts"],
```

Comment:
Why do we need to ignore this?

Try adding it to tsconfig.json like https://github.com/seamapi/react/blob/main/tsconfig.json#L46